tests/core/test_agent.py

```diff
@@ -63,30 +65,25 @@ def model_server_app(model_path: Text, model_hash: Text = "somehash"):
         assert str(x.as_dialogue()) == str(same_training_data[i].as_dialogue())
 
 
-async def test_agent_train(tmpdir, default_domain):
-    training_data_file = "examples/moodbot/data/stories.md"
-    agent = Agent(
-        "examples/moodbot/domain.yml", policies=[AugmentedMemoizationPolicy()]
-    )
-
-    training_data = await agent.load_data(training_data_file)
-
-    agent.train(training_data)
-    agent.persist(tmpdir.strpath)
-
-    loaded = Agent.load(tmpdir.strpath)
+async def test_agent_train(trained_moodbot_path: str):
```

Comment:
We currently use `Text` instead of `str`. I'd suggest stay consistent
```suggestion
async def test_agent_train(trained_moodbot_path: Text):
```